scripts/curate_basel.py

```diff
@@ -0,0 +1,120 @@
+import os
+import shutil
+import json
+import argparse
+
+
+def get_parameters():
+    parser = argparse.ArgumentParser(description='This script is curating dataset Delbono Wakehealth to BIDS')
+    parser.add_argument("-d", "--data",
+                        help="Path to folder containing the dataset to be curated",
+                        required=True)
+    parser.add_argument("-o", "--outputdata",
+                        help="Path to output folder",
+                        required=True,
+                        )
+    arguments = parser.parse_args()
+    return arguments
+
+
+def main(root_data, output_data):
```

Comment:
i would like it if we could use our [convention](https://github.com/spinalcordtoolbox/spinalcordtoolbox/wiki/Programming%3A-Naming-conventions-(variables,-files)) for variable names-- so here: `path_output`

Reply:
fixed in https://github.com/neuropoly/data-management/pull/91/commits/425d5da2a9a3c9b1596f9e9abf869850cbe595b8